docs/introduction/index.md

```diff
@@ -0,0 +1,109 @@
+
+## Blazor components
+
+Blazor apps are built with *components*. A component is a piece of UI, such as a page, dialog, or data entry form. Components can be nested, reused, and shared between projects.
+
+In Blazor, a component is a .NET class. The class can either be written directly, as a C# class (*\*.cs*), or more commonly in the form of a Razor markup page (*\*.cshtml*).
+
+[Razor](https://docs.microsoft.com/aspnet/core/mvc/views/razor) is a syntax for combining HTML markup with C# code. Razor is designed for developer productivity, allowing the developer to switch between markup and C# in the same file with [IntelliSense](https://docs.microsoft.com/visualstudio/ide/using-intellisense) support. The following markup is an example of a basic custom dialog component in a Razor file (*DialogComponent.cshtml*):
+
+```cshtml
+<div>
+    <h2>@Title</h2>
+    @RenderContent(Body)
+    <button onclick=@OnOK>OK</button>
+</div>
+
+@functions {
+    public string Title { get; set; }
+    public Content Body { get; set; }
+    public Action OnOK { get; set; }
+}
+```
+
+When this component is used elsewhere in the app, IntelliSense speeds development with syntax completion and parameter info.
+
+Components can be:
+
+* Nested.
+* Generated procedurally in code.
+* Shared via class libraries.
+* Unit tested without requiring a browser DOM.
+
+## Infrastructure
+
+Blazor offers the core facilities that most apps require, including:
+
+* Layouts
+* Routing
+* Dependency injection
+* Unit testing
+
+All of these features are optional. When one of these features isn't used in an app, the implementation is stripped out of the app when published by the IL linker.
+
+A few low-level elements are included in the framework. For example, routing and layouts aren't built-in. Routing and layouts are implemented in *user space*, code that an app developer can write without using internal APIs. These features can be replaced with different systems to suit the app's requirements. The current layouts prototype is implemented in about 30 lines of C# code, so a developer can understand and replace it if desired.
+
+## Code sharing and .NET Standard
+
+The [.NET Standard](https://docs.microsoft.com/dotnet/standard/net-standard) is a formal specification of .NET APIs that are intended to be available on all .NET implementations. Blazor supports .NET Standard 2.0 or higher. APIs that can't be supported due to browser limitations (for example, accessing the file system, opening a socket, threading, and others) throw [PlatformNotSupportedException](https://docs.microsoft.com/dotnet/api/system.platformnotsupportedexception). .NET Standard class libraries can be shared across server code and in browser-based apps.
```

Comment:
Do we call it *The* .NET Standard? I thought it was just *.NET Standard*.

Reply:
Instead of *APIs that can't be supported due to browser limitations (for example, ...* (which sounds like a defect), could we say "APIs that aren't applicable inside a web browser (for example, ..."?

Reply:
RE: "The"

This was one that I *shamelessly stole* :smile: *ROTTEN PLAGIARIZERS!* lol ... see the first line of https://docs.microsoft.com/dotnet/standard/net-standard.

A definite article here is optional from a grammatical pov. Let's drop it.